collect_app/src/main/java/org/odk/collect/android/utilities/Appearances.java

```diff
@@ -139,10 +139,10 @@ public static int getNumberOfColumns(FormEntryPrompt formEntryPrompt, ScreenUtil
                         }
                     }
                 } catch (Exception e) {
-                    Timber.e(new Error(EXCEPTION_PARSING_COLUMNS));
+                    Timber.e(new Error(EXCEPTION_PARSING_COLUMNS + columnsAppearance));
                 }
             } catch (Exception e) {
-                Timber.e(new Error(EXCEPTION_PARSING_COLUMNS));
+                Timber.e(new Error(EXCEPTION_PARSING_COLUMNS + appearance));
```

Comment:
I don't understand the structure of this code. I don't believe this outer exception can ever be triggered and it's not clear to me why there's a special exception created rather than just logging e. Still, I wanted to keep this change really narrow so I think this is good enough. Hopefully we'll be able to fix the code to address the exceptions quickly.